src/components/parameters.js

```diff
@@ -429,6 +439,36 @@ const Parameters = ({ showParameters, hideParameters, user }) => {
         );
     }
 
+    function NetworkParameters() {
```

Comment:
I think one day we should explode parameter.js to 1 component per tab (and per Internal components); but that is for another PR I think